src/components/Assistant.vue

```diff
@@ -304,27 +295,7 @@ export default {
 			await this.fetchTasks()
 		},
 		openTranslateDialog() {
-			this.displayTranslate = this.selection
-		},
-		hideTranslate() {
-			this.displayTranslate = false
-		},
-		translateInsert(content) {
-			this.$editor.commands.command(({ tr, commands }) => {
-				return commands.insertContentAt(tr.selection.to, content)
-			})
-			this.displayTranslate = false
-		},
-		translateReplace(content) {
-			this.$editor.commands.command(({ tr, commands }) => {
-				const selection = tr.selection
-				const range = {
-					from: selection.from,
-					to: selection.to,
-				}
-				return commands.insertContentAt(range, content)
-			})
-			this.displayTranslate = false
+			emit('text:translate-modal:show', { content: this.selection || '' })
```

Comment:
Does this even make sense if the selection is empty?
The previous code looks as if it did not open the modal on an empty selection:
```js
			this.displayTranslate = this.selection
```

However I just tried it on cloud.nextcloud.com and there it opened anyway but did not work (due to the broken input but also as it had no content).

Reply:
@max-nextcloud I didn't find any side effects. Yes, the text is selected fully.